packages/hoprd/README.md

```diff
@@ -21,8 +21,10 @@ Options:
   --provider                    A provider url for the Network you specified  [default: "https://still-patient-forest.xdai.quiknode.pro/f0cdbd6455c0b3aea8512fc9e7d161c1c0abf66a/"]
   --host                        The network host to run the HOPR node on.  [default: "0.0.0.0:9091"]
   --announce                    Announce public IP to the network  [boolean] [default: false]
-  --admin                       Run an admin interface on localhost:3000  [boolean] [default: false]
-  --rest                        Run a rest interface on localhost:3001  [boolean] [default: false]
+  --admin                       Run an admin interface on localhost:3000,
```

Comment:
Should be one-line. Also for ease of maintenance, the options are meant to be an exact copy of the output off `--help`.